src/bin/vip-slowlogs.ts

```diff
@@ -193,8 +193,12 @@ void command( {
 	module: 'slowlogs',
 	usage: baseUsage,
 } )
-	.option( 'limit', 'The maximum number of log entries', 500 )
-	.option( 'format', 'Output the log entries in CSV or JSON format', 'table' )
+	.option(
+		'limit',
+		'Set the maximum number of log entries. Accepts an integer value between 1 and 5000.',
+		500
+	)
+	.option( 'format', 'Output the log entries in CSV or JSON format.', 'table' )
```

Comment:
Does it make sense to list acceptable values here? We mention CSV and JSON, but by default, we get the results formatted as a table.

Reply:
Accepts "csv", "json", and "table".

(this fix for the "table" is coming soon)

Reply:
#2096

Reply:
I updated the description for the `--format` option in the `--help` menu to:
`-f, --format     Render output in a particular format. Accepts “table” (default), “csv”, and “json”.`

Does that read well?